hazelcast-client/src/main/java/com/hazelcast/client/impl/DefaultClientExtension.java

```diff
@@ -121,7 +121,7 @@ private ClientProxyFactory createClientMapProxyFactory() {
             public ClientProxy create(String id) {
                 NearCacheConfig nearCacheConfig = client.getClientConfig().getNearCacheConfig(id);
                 if (nearCacheConfig != null) {
-                    checkInMemoryFormat(nearCacheConfig.getInMemoryFormat());
+                    checkNotNATIVE(nearCacheConfig.getInMemoryFormat());
                     return new NearCachedClientMapProxy(MapService.SERVICE_NAME, id);
```

Comment:
Looks good. It just bothers me that Native is all uppercase, thus does not comply with Java convention ;)